backend/src/main/java/com/woowacourse/kkogkkog/coupon/domain/repository/CouponRepository.java

```diff
@@ -39,7 +39,7 @@ List<CouponResponse> findAllBySender(@Param("member") Member member,
         + "JOIN FETCH c.receiver "
         + "WHERE c.receiver = :member AND c.couponState.couponStatus = :status "
         + "ORDER BY c.couponState.meetingDate DESC")
-    List<CouponResponse> findAllByReceiver(@Param("member") Member member,
+    List<Coupon> findAllByReceiver(@Param("member") Member member,
                                            @Param("status") CouponStatus couponStatus);
```

Comment:
파일들에 정렬 단축키 한번만 먹여주면 좋을 것 같아요.

```suggestion
    List<Coupon> findAllByReceiver(@Param("member") Member member,
                                   @Param("status") CouponStatus couponStatus);
```